### DISCUSSION TITLE
What's the best practice to migrate masks and zones to latest 0.13 build?

### DISCUSSION THREAD

**Discussion by Mincka** [blakeblackshear/frigate#8321] [Ask A Question] [answered]
I've made my masks and zones on 0.13.0-F0C1B66 (June 3rd) and when I update my container to latest dev versions, they are completely wrong because the camera size has changed. It's unclear if that's a breaking change or something automatically migrated when coming from 0.12 to the latest dev version, or because this was changed between June and now.

I remember that I saw somewhere that the resolution was now automatically detected and if it was not, we need to specify the width and height of the camera in the `detect` section. I tried that but it did not help. I think it did not change anything because the resolution is already properly detected.

I think I'm may be stuck in the middle of two dev versions and I need to recreate all my zones and masks for 0.13, correct?

**Comment by NickM-27** [comment, marked as answer]
if you did not specifically set your width / height for your cameras then they defaulted to 1280x720 for detect and that is what the zones and masks were created with. Now in 0.13 the size is automatically detected and run on so your masks and zones will be wrong. If you were happy with how things ran before then just manually define the width/height for detect to 1280x720

**Reply by NickM-27** [reply]
at some point in the future zones and masks will be created using relative coordinates so this type of thing won't be an issue

**Reply by Mincka** [reply]
Great idea to try to force old default resolution. 👍
It will also help me to fix the coordinates programmatically by updating the old values with a ratio for the real resolution of my cameras.

Indeed, relative coordinates would be the best. 👌
